nilearn/decoding/searchlight.py

```diff
@@ -337,9 +379,7 @@ def fit(self, imgs, y, groups=None):
         )
 
         estimator = self.estimator
-        if estimator == "svc":
-            estimator = ESTIMATOR_CATALOG[estimator](dual=True)
-        elif isinstance(estimator, str):
```

Comment:
I would revert this: see my comments above about estimators and not changing the API

Reply:
@Remi-Gau 
After I added estimator as "sec" then I am getting this error
 AttributeError: 'str' object has no attribute 'fit'
 How to resolve this?

Reply:
FAILED nilearn/decoding/tests/test_searchlight.py::test_reuse_search_light_in_transform - AttributeError: 'str' object has no attribute 'fit'
FAILED nilearn/decoding/tests/test_searchlight.py::test_transform_applies_mask_correctly - AttributeError: 'str' object has no attribute 'fit'

Reply:
@Remi-Gau So why is that I am getting this error, 
FAILED nilearn/decoding/tests/test_searchlight.py::test_transform_applies_mask_correctly - AttributeError: 'str' object has no attribute 'fit' if I use estimator="svc"

Reply:
let me check